src/ripple/app/tests/HashRouter_test.cpp

```diff
@@ -28,9 +28,10 @@ namespace test {
 class HashRouter_test : public beast::unit_test::suite
 {
     void
-    testExpiration()
+    testNonExpiration()
```

Comment:
Oh, I agree - your test definitely tests what we want. I was suggesting pulling that out so it's easier to see what's going on.